pyvista/core/dataset.py

```diff
@@ -2636,7 +2687,7 @@ def find_cells_within_bounds(self, bounds: Iterable[float]) -> np.ndarray:
         if np.array(bounds).size != 6:
             raise TypeError("Bounds must be a length three tuple of floats.")
```

Comment:
must be a length six

Reply:
```suggestion
            raise TypeError("Bounds must be a length six tuple of floats.")
```

Reply:
Great suggestions/review @darikg! Feel free to make inline comments if you want. You'll get "credit" on the PR if you care about that.